lib/net/xhr.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
+const trim = ''.trim
```

Comment:
@yaslama `jsonpipe` was written in ES5 and hence doesn't have a babel gulp task to transpile ES6 to ES5.  Can you please change `const` to `var` and any other ES6 reference.